pkg/mcs/resourcemanager/server/server.go

```diff
@@ -213,7 +215,9 @@ func (s *Server) Close() {
 	}
 
 	log.Info("closing resource manager server ...")
-	s.serviceRegister.Deregister()
+	if err := s.serviceRegister.Deregister(); err != nil {
+		log.Error("failed to deregister the service", errs.ZapError(err))
```

Comment:
Do you think we should indicate which micro-service? such as
```suggestion
		log.Error("micro-service resource manager failed to deregister the service", errs.ZapError(err))
```

Reply:
The L217 prints `closing resource manager server ...`, so it doesn't matter.

Reply:
got